packages/core/core/src/Dependency.js

```diff
@@ -31,7 +32,8 @@ export function createDependency(opts: DependencyOpts): Dependency {
     md5FromString(
```

Comment:
there are a lot of things in here now. what about using `md5FromObject`? something like

```
md5FromObject({
  sourceAssetId: opts.sourceAssetId,
  moduleSpecifier: opts.moduleSpecifier,
  env: opts.env,
  target: opts.target,
  pipeline: opts.pipeline
})